sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -1631,7 +1624,7 @@ await using (var client = new EventHubClient(connectionString))
                     {
                         for (int i = 0; i < 5; i++)
```

Comment:
nit:  It'd be nice to have the 5 value set under a common variable as well, since that magic number is used in a couple of places and should match.

Reply:
Done!